src/main/java/org/folio/consortia/controller/TenantController.java

```diff
@@ -31,7 +37,18 @@ public ResponseEntity<TenantCollection> getTenants(UUID consortiumId, Integer of
 
   @Override
   public ResponseEntity<Tenant> saveTenant(UUID consortiumId, @Validated Tenant tenant) {
-    return ResponseEntity.status(CREATED).body(service.save(consortiumId, tenant));
+    FolioExecutionContext currentTenantContext = (FolioExecutionContext) folioExecutionContext.getInstance();
+    // query to tenant table to get central tenant id,
+    // if not central, throw new error
+    String centralTenantId = service.getCentralTenantId(); // central tenant id is checking for null in TenantService
```

Comment:
@SerhiiNosko does it make sense to add validation to avoid several central tenants creation?

Reply:
We discussed with Serhii, I am improving this place and remove validation for central tenant creation